src/GameEngine.js

```diff
@@ -38,6 +37,8 @@ export default class GameEngine extends Component {
   }
 
   async componentDidMount() {
+    this.timer.subscribe(this.updateHandler);
```

Comment:
For the sake of consistency - perhaps we should also migrate this line in the `GameLoop` component?